src/plugins/intel_gpu/tests/unit/fusions/reduce_fusion_test.cpp

```diff
@@ -134,28 +143,29 @@ TEST_P(reduce_eltwise_activation_quantize, basic) {
     );
 
     tolerance = 1.f;
-    execute(p);
+    execute(p, false, true);
```

Comment:
could you add a comment why this change is necessary?

Reply:
I added an explanation.